processors/config/anaMCTuple_cfg.py

```diff
@@ -33,8 +46,9 @@
 mcana.parameters["debug"] = 0
 mcana.parameters["anaName"] = "mcAna"
 mcana.parameters["partColl"] = "MCParticle"
-mcana.parameters["trkrHitColl"] = "TrackerHits"
-mcana.parameters["ecalHitColl"] = "EcalHits"
+mcana.parameters["trkrHitColl"] = "TrackerHit"
+mcana.parameters["ecalHitColl"] = "CalHit"
```

Comment:
Are these truth hit collections names changed by the simp ntupler?

Reply:
No they shouldn't be -- i ended up commenting out these collections anyways so we can revert this